### ISSUE TITLE
Only 1 tech was sold at Melnorme

### ISSUE THREAD

**Issue by Tsubasanut** [JHGuitarFreak/UQM-MegaMod#201]
Hi.
Thank you for this wonderful mod, the Hard difficulty is really nice.
I'm playing with random seed 46825 on Hard difficulty and run into some troubles.
After I bought 1st tech from Melnorme my transaction was finished with the sentence #(NEW_TECH_ALL_GONE)
And I think I've seen #(OK_BUY_NEW_TECH_12) before that, since "Shiva furnace" caught my eye. 
Maybe it's a part of the hard difficulty setting? looked through the changelog, but haven't found anything related...

Another thing - when I run into one Slylandro group orbiting Rainbow world at Beta Pegacy the game kept crashing over and over.
I'm using the latest 0.8.2 version.
I attached the save file
[uqmsave.zip](https://github.com/JHGuitarFreak/UQM-MegaMod/files/12612637/uqmsave.zip)

**Comment by Kruzen** [comment]
@Tsubasanut Hi!
Thanks for the report. It’s not a secret that Hardmode is still not polished, but we’ll investigate. Melnorme buy order breaks every version one way or another.

**Comment by JHGuitarFreak** [comment]
Can you check other Melnorme to see if they stopped selling tech as well?
In Hard Mode the various tech is spread out among the Melnorme, meaning you can't buy all tech from just one Trade Master.

As for your crash, I can confirm it indeed does crash but I don't think there is anything I can currently do to fix it.  
What I can do is ask you to turn on the Bubble Warp cheat temporarily and jump out of that solar system before the probes catch up. That seems to reset them properly.

I'm not too sure how the ship group data became corrupted. If you could go through the events before you entered Beta Pegasi maybe I can try to replicate things so I can see it get corrupted in real time.

**Comment by Tsubasanut** [comment]
Sorry, edited my message. Should have carefully read your answer.
Yes, the other Melnorme do have additional tech. So that's a feature, not a bug.

**Comment by Tsubasanut** [comment]
For the crash. I haven't done much before that. Got the portal and allied Pkunks.
I can confirm Buble Warp does not help. I warped, returned, killed 1 group of 1 and the next encounter crashed the game.
 I think only 1 of these 4 groups produces a crash. Before that save I killed 2 groups, one of them was 4-ship strong. You can see in the save I've being chased only by 2 remaining ones.
I also can confirm getting the destruct code helps - I returned later with code and cleared all 4 groups. Although when the group of 4 was self-destructed - I got the screen showing only 1 ship killed, dunno if it's important.

**Comment by JHGuitarFreak** [comment]
Yeah, it seems like somewhere down the line that particular ship group became corrupted because there's ever only supposed to be 1 probe per encounter.

And it's interesting that it kept crashing on your end even though you bubble warped out and back in. When I tried it I was able to blow up all 4 of them without a hiccup.

Though now we'll do more testing with the Probe groups in Hard Mode and make sure everything checks out.

For the record, what operating system was this on?
